modules/tensor_mechanics/src/materials/ComputeMultipleInelasticCosseratStress.C

```diff
@@ -42,6 +42,9 @@ ComputeMultipleInelasticCosseratStress::initQpStatefulProperties()
 void
 ComputeMultipleInelasticCosseratStress::computeQpStress()
 {
+  if (!_elasticity_tensor_is_constant[_qp])
+    mooseError("The Cosserat stress classes assume constant elasticity tensors");
+
```

Comment:
@WilkAndy I've changed back the inheritance structure to `ComputeMultipleInelasticStress`, but I added in this check to keep users from trying to apply a variable elasticity tensor to the Cosserat code...